app/assets/stylesheets/encounters.scss

```diff
@@ -145,11 +145,11 @@
 			}
 
 			&.open {
-				padding: 5px;
+				padding: 0.313rem;
```

Comment:
same question here. If padding grows when the root size grows, do we begin to run the risk of having no space for things?

Reply:
When would the root font size grow?